README.md

```diff
@@ -462,7 +462,8 @@ interface ScrollyBlock extends Parent {
 ```ts
 interface ScrollySection extends Parent {
 	type: "scrolly-section"
-	theme: "dark-text" | "light-text" | "dark-text-no-box" | "light-text-no-box"
+	display: "dark-background" | "light-background"
```

Comment:
[suggestion] Should these be `background-dark` and `background-light`? https://github.com/Financial-Times/n-scrollytelling/blob/main/components/image/server/utils/options.js#L10

Reply:
these values don't necessarily have to be the same as the options in `n-scrollytelling-image`. i chose this way around to mirror the old `-text` values

Reply:
I disagree with this change and think we should be using `theme` rather than a combination of `display` and `noBox`, which are trying to predict the way things are going to change in the future, and exist that way because bodyXML was so hard to change. That being said i don't care enough right now so i'm going to merge this.